lib/FrontendTool/ReferenceDependencies.cpp

```diff
@@ -31,20 +32,33 @@
 #include "llvm/Support/YAMLParser.h"
 
 using namespace swift;
+using namespace referenceDependencyKeys;
+
+template <typename Decl_T>
+static bool hasSomePrivateAccessLevel(Decl_T *decl) {
+  return decl->hasAccess() && decl->getFormalAccess() <= AccessLevel::FilePrivate;
+}
+
+/// If this declaration's privacy confines it to a file, no other file can
```

Comment:
"privacy" isn't a term we use; "access" is fine.

Reply:
Tx. Done!